locale/en_US/translations.ts

```diff
@@ -656,5 +656,15 @@
         <translation>Use generated splashscreen image as Jellyfin home background. Jellyfin will need to be closed and reopened for change to take effect.</translation>
         <extracomment>Description for option in Setting Screen</extracomment>
     </message>
+    <message>
+        <source>Cinema mode</source>
+        <translation>Cinema mode</translation>
+        <extracomment>Settings Menu - Title for option</extracomment>
+    </message>
+    <message>
+        <source>Cinema mode brings the theater experience straight to your living room with the ability to play trailers and custom intros before the main feature.</source>
+        <translation>Cinema mode brings the theater experience straight to your living room with the ability to play trailers and custom intros before the main feature.</translation>
```

Comment:
```suggestion
        <source>Cinema mode brings the theater experience straight to your living room with the ability to play custom intros before the main feature.</source>
        <translation>Cinema mode brings the theater experience straight to your living room with the ability to play custom intros before the main feature.</translation>
```

Lets not mention trailers before they're included, otherwise folks are going to ask why they're not working :)

Reply:
👍